bin/generateAcronyms.py

```diff
@@ -417,11 +417,13 @@ def find_matches_combo(filename, acronyms, ignore_str=" %"):
 
 
 def escape_for_tex(str):
-    """Escape tex nasties and return new string"""
-    nstr = str.replace("&", r"\&")
-    nstr = nstr.replace("_", r"\_")
-    nstr = nstr.replace("%", r"\%")
-    return nstr
+    """Escape tex nasties and return new string.
+    But watch out for double whamies .."""
+    text = str
+    for c in specialChars:
+        text = text.replace(c, f"\\{c}")
```

Comment:
You can use `rf` so you don't need to double escap.

```suggestion
        text = text.replace(c, fr"\{c}")
```